lib/vehicle_factory.rb

```diff
@@ -0,0 +1,14 @@
+class VehicleFactory
+    
+  def create_vehicles(data)
+    data.map do |record|
+      Vehicle.new(
+        vin: record[:vin_1_10],
+        year: record[:model_year],
+        make: record[:make],
+        model: record[:model],
+        engine: :ev
+      )
+    end
```

Comment:
Yeah! This is good. This is more what the FacilityFactory should look like. Only the data coming in that we need to create the object.